custom_dashboard/grafana_dashboards/tagging-compliance-dashboard.json

```diff
@@ -192,7 +192,7 @@
           "hide": 0,
           "includeAll": false,
           "multi": true,
-          "name": "region",
+          "name": "Region",
```

Comment:
@sdebruijn This is not required.

Reply:
Account, Project and Month all start with a capital letter. I thought I give 'region' the same treatment. 

I see that the same is true for the home-dashboard and iam-user-dashboard.

Happy to change it back if required.

Reply:
Yeah you can change it back. Thank you.

Reply:
Changed back.